tasks/install-enketo.yml

```diff
@@ -131,6 +131,7 @@
   become: true
   become_user: "root"
   shell: "killall -u {{ enketo_user }}"
+  when: not test
```

Comment:
@FrankApiyo Could we instead create the user `{{ enketo_user }}` I assume `enketo` [here](https://github.com/onaio/ansible-enketo/pull/29/files#diff-757c556599c32fa24368c139409d36d6c76ceac72ca3d1ab12486fe50c919e82R18) that would remove to have this check? Or while running the test ensuring that the `{{ enketo_user }}` matches `ansible` user?

Reply:
seems the issue was that; this command `killall` was non existent. Added it and removed `when: not test`